examples/lstm_model.py

```diff
@@ -0,0 +1,131 @@
+# Copyright © 2021 United States Government as represented by the Administrator of the National Aeronautics and Space Administration.  All Rights Reserved.
+# This ensures that the directory containing examples is in the python search directories 
+
+"""
+Example building a LSTMStateTransitionModel from data. This is a simple example of how to use the LSTMStateTransitionModel class.
+
+In this example, we generate fake data using the ThrownObject model. This is a case where we're generating a surrogate model from the physics-based model. For cases where you're generating a model from data (e.g., collected from a testbed or a real-world environment), you'll replace that generated data with your own. We then use the generated model and compare to the original model.
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+from prog_models.lstm_model import LSTMStateTransitionModel
+from prog_models.models import ThrownObject
+
+def run_example():
+    # -----------------------------------------------------
+    # Example 1- set timestep 
+    # Here we will create a model for a specific timestep.
+    # The model will only work with that timestep 
+    # This is useful if you know the timestep you would like to use
+    # -----------------------------------------------------
+    TIMESTEP = 0.01
+
+    # Step 1: Generate data
+    # We'll use the ThrownObject model to generate data.
+    # For cases where you're generating a model from data (e.g., collected from a testbed or a real-world environment), 
+    # you'll replace that generated data with your own.
+    print('Generating data')
+    m = ThrownObject()
+
+    def future_loading(t, x=None):
+        return m.InputContainer({})  # No input for thrown object 
+
+    data = m.simulate_to_threshold(future_loading, threshold_keys='impact', save_freq=TIMESTEP, dt=TIMESTEP)
+
+    # Step 2: Generate model
+    # We'll use the LSTMStateTransitionModel class to generate a model from the data.
+    print('Building model...')
+    m2 = LSTMStateTransitionModel.from_data(
+        (data.inputs, data.outputs),  
+        sequence_length=4, 
+        epochs=250,
```

Comment:
What does epochs gets 250 here mean? I assuming that this is part of the LSTMStateTransitionModel?